### PR TITLE
Corrections to function in Zikula 1.4

### PR DESCRIPTION
**Pull request by nmpetkov**
Corrections:
- File type declarations.
- Some handling not set values.

Related: #18 - now installation passes.

### CODE REVIEW

File: lib/AdvancedPolls/Entity/Desc.php [hunk 1]
```diff
@@ -57,9 +57,9 @@ class AdvancedPolls_Entity_Desc extends Zikula_EntityAccess
     /**
      * The following are annotations which define the description field.
      *
-     * @ORM\Column(type="text", nullable="true")
```
Comment (cmfcmf) [inline lib/AdvancedPolls/Entity/Desc.php]: Why did you remove it completely instead of removing the quotes only, like below?

Comment (cmfcmf) [inline lib/AdvancedPolls/Entity/Desc.php]: You may need to re-add the `= null` in the line below, no?

Comment (nmpetkov) [inline lib/AdvancedPolls/Entity/Desc.php]: Yes, done.

**Comment by cmfcmf** [comment]
Thank you @nmpetkov!
